### PR TITLE
convert docx to markdown

### PR DESCRIPTION
**Pull request by mayank922**
Task #1068

### CODE REVIEW

File: dev_scripts/documentation/Hiring_process.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+- If we receive a single candidate application and find no red flags in the
+  profile, we should proceed further in the process
+  - It's ok to ask more team members to take a look
+
+- If the candidate is a no-go, GP sends an email of rejection
+
+## Mentors
+
+- Hiring Meisters (e.g., Sonaal and Samarth) supervise the hiring process
+  - ```Docs/onboarding/sorrentum.hiring_meister.how_to_guide.md```
```
[the remark this answers is not in the document]
Comment (mayank922) [reply dev_scripts/documentation/Hiring_process.md:67]: Hi @gpsaggese 

Are you referring to this file?
https://github.com/kaizen-ai/kaizenflow/blob/master/docs/onboarding/kaizenflow.hiring_meister.how_to_guide.md

[the remark this answers is not in the document]
Comment (samarth9008) [reply dev_scripts/documentation/Hiring_process.md:67]: Yes

File: dev_scripts/documentation/Hiring_process.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
+
```
Comment (gpsaggese) [inline dev_scripts/documentation/Hiring_process.md:1]: The PR name doesn't follow our convention.

There is a target to generate the PR names automatically
```
work_tools/all.git.how_to_guide.md:98:    `i gh_issue_title`
```

Ok not to rename this one, but let's use the right one moving fwd

Comment (mayank922) [reply dev_scripts/documentation/Hiring_process.md:1]: Got it.

File: dev_scripts/documentation/Hiring_process.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+- If we receive a single candidate application and find no red flags in the
+  profile, we should proceed further in the process
+  - It's ok to ask more team members to take a look
+
+- If the candidate is a no-go, GP sends an email of rejection
+
+## Mentors
+
+- Hiring Meisters (e.g., Sonaal and Samarth) supervise the hiring process
+  - [Docs/onboarding/kaizenflow.hiring_meister.how_to_guide.md](https://github.com/kaizen-ai/kaizenflow/blob/master/docs/onboarding/kaizenflow.hiring_meister.how_to_guide.md)
```
Comment (gpsaggese) [inline dev_scripts/documentation/Hiring_process.md:67]: we represent links as [docs/...](/docs/...)

See documentation

File: dev_scripts/documentation/Hiring_process.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+- We use Asana tasks to communicate about the candidates
+
+- There are multiple pipeline stages on Asana that candidates go through
+  - **Candidates**
+    - Anybody who wants to join our project/company
+  - **On-boarding on KaizenAI**
+    - _Goal_: set up candidates to work on their laptop on KaizenAI
+    - _Duration_: few days
+    - The checklist for onboarding on KaizenAI is
+      [here](https://github.com/kaizen-ai/kaizenflow/blob/master/docs/onboarding/kaizenflow.hiring_meister.how_to_guide.md#on-boarding-bug-creation)
```
Comment (gpsaggese) [inline dev_scripts/documentation/Hiring_process.md:88]: relative path

Comment (mayank922) [reply dev_scripts/documentation/Hiring_process.md:88]: Hi @gpsaggese 

You want me to remove "here" in the file and add a path with the link right?

Something like this :
[docs/onboarding/kaizenflow.hiring_meister.how_to_guide.md#on-boarding-bug-creation](https://github.com/kaizen-ai/kaizenflow/blob/master/docs/onboarding/kaizenflow.hiring_meister.how_to_guide.md#on-boarding-bug-creation)

Comment (samarth9008) [reply dev_scripts/documentation/Hiring_process.md:88]: Something like this

```
[here](/docs/onboarding/kaizenflow.hiring_meister.how_to_guide.md#on-boarding-bug-creation)
```

**Pull request merged**